### ISSUE TITLE
NVDA fails to notify typed misspelled words in MS Edge Chromium via the buzz

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#11480]
If the "Spelling errors" check box is checked under the Document Formatting tab of the Settings dialog, NVDA doesn't play the buzz which is associated with the typing of misspelled words in Ms Edge Chromium 85. Quite interestingly, it works with Google Chrome.
### Steps to reproduce:
1. With MS Edge Chromium 84 open a page which has a form -- like the following: https://cloud.google.com/text-to-speech
2. Type some words in the second form -- inside the frame, making sure a few of them are misspelled.
### Actual behavior:
NVDA doesn't play the buzz as soon as you hit space after the misspelled word.
### Expected behavior:
NVDA should immediately play the buzz as soon as you hit space after typing the misspelled word.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Alpha-20685
#### Windows version:
Win 10 Pro 64-Bit, Version 2004
#### Name and version of other software in use when reproducing the issue:
MS Edge Chromium 84
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No. However -- and in case it might help, the issue is limited to Edge Chromium as it works well with Google Chrome. Also haven't tested Firefox as I don't have it, but it works well with Thunderbird 78.1.1.
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes

**Comment by amirsol81** [comment]
@seanbudd @michaelDCurran @leonardder Unfortunately both Chrome/Edge 112 exhibit this behavior. So it seems that the chime notification for misspelled words is totally broken in Chrome/Edge. I've updated the title of the issue to reflect this.

**Comment by amirsol81** [comment]
@Qchristensen @seanbudd @michaelDCurran Any chance of looking into this? It's still effectively broken with Chrome V115.

**Comment by Qchristensen** [comment]
I don't have a resolution for this directly, but one workaround worth considering is Tiny Spell, a program which runs in the background and gives you spell checking in ANY program, even those with no spelling functionality at all, such as NotePad: https://www.tinyspell.com/
